docs/assemblies/assembly_packaging-types.adoc

```diff
@@ -0,0 +1,16 @@
+
+[id='packaging-types_{context}']
+= Packaging Types
+
+:parent-context: {context}
+
+When using {Thorntail}, there are the following ways to package your
+runtime and application, depending on how you intend to use and deploy
+it:
```

Comment:
Single sentence, single line?

Reply:
Certainly, thanks for catching that. Somehow, a few files have the line breaks still.